### ISSUE TITLE
Interact with Alcatraz via command line

### ISSUE THREAD

**Issue by fpg1503** [supermarin/Alcatraz#332]
Alcatraz should have command line tools that allow commands like
`alcatraz upgrade`
`alcatraz search package`
`alcatraz install package`
`alcatraz update package`
`alcatraz remove package`

**Comment by cannyboy** [comment]
+1. Useful for scripting a Dev Stack setup.

**Comment by jurre** [comment]
Yes this is a great idea :) I know @neonichu has started working on this a while ago

**Comment by neonichu** [comment]
Yah, I'm still planning to make this at some point :tm:

**Comment by jkyin** [comment]
Good idea.

**Comment by wczekalski** [comment]
I think it could also include `alcatraz reset-blocked` command to reset blocked plugins as in https://github.com/supermarin/Alcatraz/pull/298

**Comment by iotize** [comment]
Adding interest for this. :+1:

**Comment by neonichu** [comment]
There's https://github.com/neonichu/azkaban
